apps/courses/migrations/0001_squashed_0002_auto_20180625_1547.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this is what you were looking for @sampaccoud ?

Reply:
I would have deleted and recreated 0001 since we have nothing in operation yet but what you did should be better.